packages/@aws-cdk-testing/framework-integ/test/aws-iam/test/integ.managed-policy.ts

```diff
@@ -24,7 +24,8 @@ const policy3 = ManagedPolicy.fromAwsManagedPolicyName('SecurityAudit');
 user.addManagedPolicy(policy3);
 
 const role = new Role(stack, 'Role', { assumedBy: new AccountRootPrincipal() });
-role.grantAssumeRole(policy.grantPrincipal);
+role.grantAssumeRole(user);
```

Comment:
Why is this change needed in the test file?

Reply:
I think to prove you can still use `grantAssumeRole` on an IAM user.

Reply:
This was a preexisting fault that bubbled up when I made this change.  The error was something to the effect that you cannot grant a policy assume permission.  Since the apparent intent of this portion was to prove that grantAssume could be given to the policy's principal (the user) I changed it to that.

Reply:
I'm taking another look at this to see if I can reproduce it

Reply:
I can't get this to reproduce.  I've reverted the change and re-run the snapshot.  Strange.  Pushing an update.